frontend/src/metabase/components/Detail.css

```diff
@@ -3,12 +3,13 @@
     --subtitle-color: #AAB7C3;
     --muted-color: #DEEAF1;
     --blue-color: #2D86D4;
+    --icon-width: calc(48px + 1rem);
```

Comment:
The alternative I had before was combining a pixel based padding and rem based margin to achieve the same layout, which feels much worse in my opinion, both in terms of readability and flexibility for future layout changes. Definitely open to hearing about other possible alternatives though.